test/image/mocks/fill_trace.json

```diff
@@ -0,0 +1,52 @@
+{
+  "data": [
```

Comment:
Would you mind renaming this mock `gl2d_fill_trace.json`? We like to prefix gl-based mocks by either `gl2d_` or `gl3d_`.

Would should probably write down in the CONTRIBUTING file at some point.

Reply:
Good point for this PR - but rather than describing this in `CONTRIBUTING.md` how about we make the image test system support subfolders and give them a little more organization? It's getting impossible to find what you want in `mocks` and `baselines`...